### ISSUE TITLE
Performance audit

### ISSUE THREAD

**Issue by marc-bouvier** [iroco-co/audits-frago#6]
Run and publish lighthouse performance test for iroco.co

**Comment by marc-bouvier** [comment]
```shell

sudo npm install -g lighthouse
mkdir -p  contents/audits/iroco-landing/lighthouse
lighthouse https://iroco.co --output-json --output-path=./contents/audits/iroco-landing/lighthouse/$(date -u +%Y-%m-%d).json --no-enable-error-reporting


  LH:ChromeLauncher:error connect ECONNREFUSED 127.0.0.1:37753 +2ms
  LH:ChromeLauncher:error Logging contents of /tmp/lighthouse.89VRCwZ/chrome-err.log +1ms
  LH:ChromeLauncher:error
  LH:ChromeLauncher:error Command '/usr/bin/chromium-browser' requires the chromium snap to be installed.
  LH:ChromeLauncher:error Please install it with:
  LH:ChromeLauncher:error
  LH:ChromeLauncher:error snap install chromium
  LH:ChromeLauncher:error
  LH:ChromeLauncher:error  +0ms
Unable to connect to Chrome
~/s/a/sandbox ❯❯❯  snap install chromium                                                                ✘ 1

erreur : cannot communicate with server: Post "http://localhost/v2/snaps/chromium": dial unix /run/snapd.socket: connect: no such file or directory
~/s/a/sandbox ❯❯❯



```
